command/ca/acme/eab/add.go

```diff
@@ -0,0 +1,94 @@
+package eab
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	adminAPI "github.com/smallstep/certificates/authority/admin/api"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/utils/cautils"
+	"github.com/urfave/cli"
+	"go.step.sm/cli-utils/errs"
+)
+
+func addCommand() cli.Command {
+	return cli.Command{
+		Name:   "add",
+		Action: cli.ActionFunc(addAction),
+		Usage:  "add ACME External Account Binding Key",
+		UsageText: `**step beta ca acme eab add** <provisioner> [<reference>]
+[**--admin-cert**=<file>] [**--admin-key**=<file>]
+[**--admin-provisioner**=<string>] [**--admin-subject**=<string>]
+[**--password-file**=<file>] [**--ca-url**=<uri>] [**--root**=<file>]
+[**--context**=<name>]`,
+		Flags: []cli.Flag{
+			flags.AdminCert,
+			flags.AdminKey,
+			flags.AdminProvisioner,
+			flags.AdminSubject,
+			flags.PasswordFile,
+			flags.CaURL,
+			flags.Root,
+			flags.Context,
+		},
+		Description: `**step beta ca acme eab add** adds ACME External Account Binding Key.
+
+## POSITIONAL ARGUMENTS
+
+<provisioner>
+: Name of the provisioner to add an ACME EAB key to 
```

Comment:
Name of the provisioner to which the ACME EAB key will be added.

Reply:
Fixed in 686e78f8613be934418156b4c357eacb32e06f80.